src/components/external/DisputeManagement/components/DisputesData/DisputeData.tsx

```diff
@@ -119,12 +121,15 @@ export const DisputeData = ({
         setFlowState('accept');
     }, [dispute, setDispute, setFlowState]);
 
-    if (!dispute || isFetching) {
-        return <DataOverviewDetailsSkeleton skeletonRowNumber={5} />;
-    }
+    const onDefendClick = useCallback(() => {
+        dispute && setDispute(dispute);
+        if (dispute?.defensibility === 'defendable') {
+            setFlowState('defendReasonSelectionView');
+        }
+    }, [dispute, setDispute, setFlowState]);
```

Comment:
I don't think we need to `setDispute` explicitly here, since that already happens in the `onSuccess` callback of `useFetch` above. This also applies to the `onAcceptClick` callback.

Also, I'd suggest replacing the condition here with the `isDefendable` value. Similar can be done in the `onAcceptClick` callback.

```suggestion
    const onDefendClick = useCallback(() => {
        if (isDefendable) {
            setFlowState('defendReasonSelectionView');
        }
    }, [setFlowState]);
```